app/source/components/stats-table/stats-table.jsx

```diff
@@ -68,13 +67,13 @@ class StatsTable extends React.Component {
       {months &&
         this.lastTwelveMonths().map((month, index) => (
           <React.Fragment key={month}>
-            {index === currentMonth + 1 && <th className="spacer" />}
+            {index === 11 - currentMonth && <th className="spacer" />}
```

Comment:
I have no clue what this index logic does. `index === 11 - currentMonth` should probably be extracted to some function with an understandable name. That way the logic also won't have to be duplicated, reducing the chance for bugs in the future

Reply:
I agree, I have extracted this logic into some functions attempting to make things clearer